youtube_dl/extractor/liveleak.py

```diff
@@ -120,6 +120,19 @@ def _real_extract(self, url):
             }
 
         for idx, info_dict in enumerate(entries):
+
+            for f in info_dict['formats']:
+                match = re.match(r'https?://(?:\w+\.)?liveleak\.com/.*?\.(\w+)(\.\w+\.mp4)(?:\?.+)?', f['url'])
+                if match:
+                    ext = match.group(1)
+                    new_url = f['url'].replace(match.group(2), '')
+                    info_dict['formats'].append({
+                        'url': new_url,
+                        'ext': ext,
+                        'format_note': 'original, without watermark',
+                        'format_id': '%s-orig' % (f['format_id'] or 'default')
+                    })
```

Comment:
1. This will duplicate source format when multiple non source formats are available.
2. Source format must be named `source`.